tool/ExcelData/Cli/DataHelperCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Datask.Tool.ExcelData;
+
+[Command("data-helper", "d")]
+[CommandHelp("Generates data helper file with excel table and column information.")]
+public sealed class DataHelperCommand : BaseCommand
+{
+    [Argument(Order = 0)]
+    [ArgumentHelp("file name", "The name of the config file to create data helper class.")]
+    public FileInfo ConfigFile { get; set; } = null!;
+
+    protected override async Task<int> ExecuteAsync(StatusContext ctx, IParseResult parseResult)
+    {
+        //Parse config file and fill the data model
+        DataHelperConfiguration? config = ParseConfiguration();
+        if (config is null || config.Flavors is null || config.Flavors.Count == 0)
+        {
+            AnsiConsole.MarkupLine($"The config does not contains valid value..");
+            return 0;
+        }
+
+        if (string.IsNullOrEmpty(config.FilePath))
+            config.FilePath = Path.Combine(Directory.GetCurrentDirectory(), "TestDataHelper.cs");
+
+        DataExtensionBuilder builder = new(config);
+
+        builder.OnStatus += (_, args) =>
```

Comment:
There should be no need to call `builder.OnStatus`. This is already done in the `BaseCommand`. Just use the `ctx` parameter directly to set status.